HACKING.md

```diff
@@ -328,9 +328,9 @@ If the above systemd-sysext approach does not work for you, you can also test
 changes with some bind mounts.
 
 To test changes to the login page, you can bind-mount the build tree's
-`dist/static/` directory over the system one:
+`dist/` directory over the system one:
 
-    sudo mount -o bind dist/static/ /usr/share/cockpit/static/
+    sudo mount -o bind dist /usr/share/cockpit
 
 Likewise, to test changes to the branding, use:
```

Comment:
We should probably remove the special branding par too, as it'd be in dist anyway?

Reply:
No, branding is separate, that needs to stay.

Reply:
That said, you'd need to create the target directory first.

Reply:
Updated accordingly.